atecc608a/main.c

```diff
@@ -76,10 +76,22 @@
     "the slot now behaves according to the policies set by the associated\n"\
     "configuration zone’s values. [y/n]: "
 
-/* Data used by tests */
+/* Physical locations of the keys in the secure element. */
 psa_key_slot_number_t atecc608a_private_key_slot = 1;
 psa_key_slot_number_t atecc608a_public_key_slot = 9;
 
+/* Key IDs used by applications. Key ID values are often different from the
+ * physical location of the key in the secure element.
+ *
+ * Note: For this example, we use IDs that are outside the range of valid
```

Comment:
I find this paragraph a bit confusing, like the previous one.

> To avoid accidental confusion between slot numbers (physical location) and key IDs (logical name), we pick key IDs that are outside of the numerical range of slot numbers supported by the secure element, which is 0..15.